### ISSUE TITLE
testee fails to start on Travis CI with Node 12

### ISSUE THREAD

**Issue by bmomberger-bitovi** [bitovi/testee#187]
Node 12.0.0 was released yesterday (2019-04-23) and since then, all CanJS build jobs that do not pin the Node version in Travis config are failing when testee starts.  Here's an example error message from running testee on can-attribute-observable's test suite:

```
ReferenceError: primordials is not defined
    at fs.js:27:35
    at req_ (/home/travis/build/canjs/can-attribute-observable/node_modules/natives/index.js:143:24)
    at Object.req [as require] (/home/travis/build/canjs/can-attribute-observable/node_modules/natives/index.js:55:10)
    at Object.<anonymous> (/home/travis/build/canjs/can-attribute-observable/node_modules/fstream/node_modules/graceful-fs/fs.js:1:37)
    at Module._compile (internal/modules/cjs/loader.js:759:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:770:10)
    at Module.load (internal/modules/cjs/loader.js:628:32)
    at Function.Module._load (internal/modules/cjs/loader.js:555:12)
    at Module.require (internal/modules/cjs/loader.js:666:19)
    at require (internal/modules/cjs/helpers.js:16:16)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! can-attribute-observable@1.2.4 testee: `testee test/test.html --browsers firefox`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the can-attribute-observable@1.2.4 testee script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2019-04-24T00_06_13_248Z-debug.log
npm ERR! Test failed.  See above for more details.
The command "npm test" exited with 1.
```

This is a probable `npm ls` route leading to this problem:
```
└─┬ testee@0.8.1
  └─┬ miner@0.2.1
    └─┬ browserstacktunnel-wrapper@1.4.2
      └─┬ unzip@0.1.11
        └─┬ fstream@0.1.31
          └─┬ graceful-fs@3.0.11
            └── natives@1.1.6 
```

**Comment by bmomberger-bitovi** [comment]
`natives` is deprecated and its deprecation message says to use `graceful-fs` version 4 or later.  This is likely just caused by outdated dependencies.  I'm investigating whether we can upgrade miner and avoid this problem.

**Comment by chasenlehara** [comment]
This was fixed in [0.9.1](https://github.com/bitovi/testee/releases/tag/v0.9.1), right? Would you mind writing the release notes please?
